src/TableView.elm

```diff
@@ -314,9 +318,13 @@ viewCell dateWithSessions model timeDelimiters numColumns timeDelimiter index co
                 |> Maybe.withDefault -1
 
         trackId =
-            sessionStarting
-                |> Maybe.map .trackId
-                |> Maybe.withDefault 0
+            case (Maybe.map .trackId sessionStarting) of
```

Comment:
Agree about duplication. I'm confused about how it would work with andThen. I've tried but it's not working :( I think the issue may be because we return 0 for the Nothing case but andThen returns Nothing for the Nothing case...